src/tests/resources/providersConfig.ts

```diff
@@ -0,0 +1,130 @@
+import Providers from '../../providers';
+
+const providersConfig: Record<keyof typeof Providers, any> = {
```

Comment:
Lets strictly define type here.
ProviderTestConfig
Type ProviderTestConfig
name : keyof typeof Providers,
provider/ApiKey,
operations: LlmOperation (defined below)
skip?: boolean (if they want to skip this call)

type LlmOperation
type: 'completions' | 'chatCompletions' | 'proxy'
operation: object